howtos/misc/_posts/2021-12-06-178-and-searchbar.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title:  "PrestaShop 1.7.8 and the search bar issue"
```

Comment:
```suggestion
title:  "How to solve the search bar after upgrading to PrestaShop 1.7.8"
```

Reply:
@eternoendless "How to solve the search bar" doesn't sound good, you meant "How to solve an issue with the search bar" or "How to solve the search bar display"?

Reply:
Yes, my guess is that @eternoendless wants people to understand this content is about solving the issue, so the importants words are "how" and "solve", the rest can be explored

Reply:
Sorry, my mistake, forgot a word. I chose an SEO friendly wording
```suggestion
title:  "How to solve the search bar issue after upgrading to PrestaShop 1.7.8"
```